README.md

```diff
@@ -214,7 +214,9 @@ import (
 // ...
 var _ = BeforeSuite(func() {
 	// block all HTTP requests
-	httpmock.ActivateNonDefault(resty.DefaultClient.GetClient())
+	httpmock.ActivateNonDefault(resty.New().GetClient())
```

Comment:
I do not use resty, but should'nt the `resty.New()` instance be kept to use it later in tests?

Reply:
Hm.. may be more efficient but should not interfere with different tests as well.
How to store it here?